root/etc/s6-overlay/s6-rc.d/init-mod-code-server-rust-install/run

```diff
@@ -26,13 +29,6 @@ if [[ -f "$RUST_BINARY_ARCHIVE_PATH" ]]; then
   echo "**** installing rust ****"
   "$RUST_BINS_PATH/install.sh"
   rm -rf "$RUST_BINS_PATH"
-
-  echo "**** adding build tools ****"
-  if grep_os_release "debian"; then
-    echo "build-essential" >>"$PACKAGES_TO_INSTALL_LIST"
-  else
-    echo "gcc" >>"$PACKAGES_TO_INSTALL_LIST"
-  fi
 else
-  echo "**** rust already installed ****"
+  echo "**** rust bin does not exist ****"
```

Comment:
Last nitpicky comment I promise :-)
While this log message is technically correct from a coding perspective, it would be confusing to an end user. Can you change it to something like `latest rust is already installed`? Currently it sounds like something unexpected happened whereas it really means no install or update is necessary.

Reply:
No problem 
I am thankful for your detailed explanation and resilience :smile: